### PR TITLE
Synth updates

### PR DESCRIPTION
**Pull request by RAIST5150**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -1323,7 +1323,7 @@ INSERT INTO `synth_recipes` VALUES (19920,1,0,0,23,0,0,0,0,0,0,4100,4242,16704,0
 -- INSERT INTO `synth_recipes` VALUES (19926,1,0,0,255,0,0,0,0,0,0,4100,4242,12816,0,0,0,0,0,0,0,649,817,818,850,2,6,1,2,'Scale Cuisses (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19927,1,0,0,255,0,0,0,0,0,0,4100,4242,16466,0,0,0,0,0,0,0,707,651,651,651,1,1,1,1,'Knife (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19928,1,0,0,255,0,0,0,0,0,0,4100,4242,16631,0,0,0,0,0,0,0,895,801,651,651,1,1,2,2,'Kaiser Sword (desynth)');
--- INSERT INTO `synth_recipes` VALUES (19929,1,0,0,255,0,0,0,0,0,0,4100,4242,16475,0,0,0,0,0,0,0,653,653,653,653,1,1,1,1,'Mythril Kukri (desynth)');
+INSERT INTO `synth_recipes` VALUES (19929,1,0,0,42,0,0,0,0,0,0,4100,4242,16475,0,0,0,0,0,0,0,853,716,653,653,1,1,1,1,'Mythril Kukri (desynth)'); -- on synth level, used XI results
```
Comment (TeoTwawki) [inline sql/synth_recipes.sql]: comment unclear. "on synth level"? meaning desynth lv based on the synth lv? "used XI results", meaning ffxiclopedia?

Comment (RAIST5150) [reply sql/synth_recipes.sql]: Sorry yeah... was trying to keep the comments shorter.

Need me to edit and resubmit tonight?

Comment (TeoTwawki) [reply sql/synth_recipes.sql]: expand them out just a little bit so we can tell the meaning at a glance. XI could be easily mistaken for other things. "used synth lv" would suffice and is only 2 more characters for that part

**Comment by RAIST5150** [comment]
good point... I should have caught that.  My bad... mostly copy/pasta from my notes.
Didn't notice some had some extraneous comments not really needed too, so cleaned them up a bit as well.

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**